File: PUSGS_Project/Core/Entities/RentACar.cs
using System.Collections.Generic;

namespace Core.Entities
{
    public class RentACar
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public string Description { get; set; }
        public ICollection<Car> Cars { get; set; }
        public ICollection<Branch> Branches { get; set; }

        public ICollection<Rating> Ratings { get; set; }

        public RentACar()
        {
            Cars = new HashSet<Car>();
            Branches = new HashSet<Branch>();
            Ratings = new HashSet<Rating>();
        }
    }
}

File: PUSGS_Project/ClientApp/src/app/entities/rent-a-car.ts
import { Car } from "./car";
import { Branch } from "./branch";

export class RentACar {
  id: number;
  name: string;
  address: string;
  description: string;
  // TODO: cenovnik
  cars: Car[];
  branches: Branch[];
  ratings: number[];
  averageRating: number;
}
